packages/mds-policy-author/tests/api.spec.ts

```diff
@@ -96,7 +96,12 @@ describe('Tests app', () => {
 
     it('tries to post invalid policy', done => {
       const bad_policy_json: Policy = clone(POLICY_JSON_WITHOUT_PUBLISH_DATE)
+
+      /* eslint-reason intentionally bad test */
+      /* eslint-disable-next-line @typescript-eslint/ban-ts-comment */
+      // @ts-ignore
```

Comment:
Why? This seems like a good case to test.

Reply:
The test is fine, I meant it's intentionally testing a "bad data" case. Linter yells when mutating an object and deleting a non-optional property. The test is doing something that would never happen in a fully-typed context.

Reply:
Ok, I do hate being that person, but I feel like given that Michael and I both had reactions to these commented out lines, a little bit more explanation is required.

Reply:
Updated in [this commit](https://github.com/lacuna-tech/mds-core/pull/427/commits/129d48096d7c67b8900867db9c11daf648e9a4dd)